tools/material-ci/angular_material_test_blocklist.js

```diff
@@ -763,23 +763,23 @@ window.testBlocklist = {
   },
   "MatMenu animations should enable ripples on items by default": {
     "error": "TypeError: Cannot read property 'query' of null",
-    "notes": "Unknown"
+    "notes": "FW-1059: Diagnose root causes of failures in MatMenu"
```

Comment:
I think you pasted the wrong summary? Seems like it should be:

```suggestion
    "notes": "FW-1059: DebugNode.query should query nodes in the logical tree"
```

Reply:
Ops, not sure how that happened. Fixed.